PhysicsTools/PatAlgos/python/producersHeavyIons/heavyIonJets_cff.py

```diff
@@ -72,6 +72,8 @@
 
 from Configuration.ProcessModifiers.run2_miniAOD_pp_on_AA_103X_cff import run2_miniAOD_pp_on_AA_103X
 from RecoJets.JetProducers.ak4PFJets_cfi import ak4PFJets,ak4PFJetsCHS
+ak4PFJets.src = "pfEmptyCollection"
+ak4PFJetsCHS.src = "pfEmptyCollection"
```

Comment:
I'd expect `run2_miniAOD_pp_on_AA_103X.toModify` and not here, but in the original file ak4PFJets_cfi

Reply:
If I use run2_miniAOD_pp_on_AA_103X, the input collection name is not modified in patTuple_PF2PAT_cfg.py.  This is true whether the toModify statement is in the current location or in the original file.

Reply:
well, OK; the point is that this last commit with ` ak4PFJets.src = "pfEmptyCollection" ` is not appropriate.
Direct modifications of values in secondary files should be avoided.

Here this modification changes the default miniAOD configuration to have e.g. `ak4PFJets.src = "pfEmptyCollection"`, which is clearly wrong.